Taskfile.yml

```diff
@@ -112,7 +113,7 @@ tasks:
       GOOS: darwin
       GOARCH: amd64
     vars: *build-vars
-  
+
   install-lint:
```

Comment:
Should I do the same here for installation of `golangci-lint`?
```
- go install github.com/golangci/golangci-lint/cmd/golangci-lint@v$GOLANGCI_VERSION \
- go clean -cache -modcache
```

Reply:
I'd say 'yes' , I don't remember the reason behind this.